src/components/project/ProjectIntroForm.tsx

```diff
@@ -64,15 +66,16 @@ const ProjectIntroForm: React.FC<{
     updateProjectCover({
       variables: {
         projectId: project.id,
-        previewUrl: `https://${process.env.REACT_APP_S3_BUCKET}/project_covers/${appId}/${project.id}?t=${uploadTime}`,
+        previewUrl: `https://${process.env.REACT_APP_S3_BUCKET}/project_covers/${appId}/${project.id}/${coverId}?t=${uploadTime}`,
         coverUrl:
           project.coverUrl === null
-            ? `https://${process.env.REACT_APP_S3_BUCKET}/project_covers/${appId}/${project.id}?t=${uploadTime}`
+            ? `https://${process.env.REACT_APP_S3_BUCKET}/project_covers/${appId}/${project.id}/${coverId}?t=${uploadTime}`
```

Comment:
we don't need the `uploadTime` if we have used the `uuid`

Reply:
fixed